.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+
+jobs:
+  tests:
+    docker:
+      - image: ubuntu:18.04
```

Comment:
Some examples of GitHub Pages [deployment for circleci](https://circleci.com/blog/deploying-documentation-to-github-pages-with-continuous-integration/) show using the `node` docker image as a base. Would that make more sense than unboontoo?

Reply:
The main reason I prefer `ubuntu 18.04` is because my local desktop is also running this Linux distribution. Using a docker image of different distribution may make the debugging harder.

Reply:
Perhaps, but the node docker based on debian won't be too different. I think it's ideal to minimize the amount of extra installation we have to do on top of the base image. These extra steps slow down the tests, create extra outputs, and extra steps making the build more dependent on third-party resources.

Should we try the node docker and go back to ubuntu if that creates problems?

Reply:
I've changed the primary docker container into `node:8.10.0`.